src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,231 @@
+package duke;
```

Comment:
Perhaps you can divide into multiple packages 😃

Reply:
Haha will take note